### DISCUSSION TITLE
After the acount username changed, The repository [name] has been retired and cannot be reused

### DISCUSSION THREAD

**Discussion by XuHandsome** [community/community#23044] [New to GitHub]
<p>I have two Github accounts, A and B. The code warehouse “kxs” is in account A</p>
<ol>
<li>Change A account username to C</li>
<li>Change B account username to A</li>
<li>Create code repository “kxs” for the new account A, I get an error:</li>
</ol>
<ul>
<li>“The repository kxs has been retired and cannot be reused”</li>
<li>Other repository name in old account A also this</li>
</ul>

**Comment by XuHandsome** [comment]
<p><strong>like this:</strong><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/f/cf894384fae092d66441d85a998f6e2414a183cc.png" data-download-href="/uploads/short-url/tBWVcuWLfTqkMvqitGiUz30iQtC.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/21697438/181074513-6ea38686-ba3b-454c-9fd9-358771ccac42.png" alt="image" data-base62-sha1="tBWVcuWLfTqkMvqitGiUz30iQtC" width="690" height="225" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/f/cf894384fae092d66441d85a998f6e2414a183cc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">721×236 15.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p>yup that’s quite tricky,</p>
<p>you want try requesting it but there is no assurance</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/contact/account">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/47092464/181074517-8d1f2110-49fa-40a0-bc87-b31e5ba975c6.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/contact/account" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/contact/account" target="_blank" rel="noopener">Accounts - GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by XuHandsome** [comment]
<p>thx, I’ll try it，I’ve seen other users have the same problem</p>

**Comment by billythekid** [comment]
<p>Hi folks,</p>
<p>This is likely down to the <a href="https://github.blog/2018-04-18-new-tools-for-open-source-maintainers/">Popular repository namespace retirement</a> feature.</p>
<p>Cheers,<br>
Billy :cowboy_hat_face:</p>

**Comment by queenofcorgis** [comment]
<p>Hi <a class="mention" href="/xuhandsome">@XuHandsome</a> checking in here were <a class="mention" href="/billythekid">@billythekid</a> or <a class="mention" href="/jdevstatic">@jdevstatic</a> able to solve this for you? If so, don’t forget to mark the solution :+1:t3:</p>

**Comment by ghost** [comment]
<p>That error is really becoming annoying i had the same problem but the support team didn’t reply even a single dot on my request, even if you own the previous username vice-versa a lot of my friends having the same problem as this.</p>

**Comment by jfullstackdev** [comment]
<p>it’s highly discouraged to change your username, as it is just like your identity that cannot be changed from time to time, it’s part of the URL so problems will surely arise</p>

**Comment by fulplan** [comment]
<p><strong>Github Account Name</strong> : fulplan <a href="https://github.com/fulplan" class="inline-onebox" rel="noopener nofollow ugc">fulplan (FulPlan Mutasim) · GitHub</a>**</p>
<p>Actually, I recently changed my username from fulplan to a different name, and I also changed my special repo name on GitHub.</p>
<p>After some days I started finding trouble as all because everywhere I used my previous username So I changed again to my previous one, but there is an error on GitHub Repository name. So I accidentally rename my special repository.<br>
Now when I try to make new repo there error message pop up<br>
‘The repository fulplan has been retired and cannot be reused.’</p>
<p>Can you please fix this asap?</p>
<p>Regards,</p>
<p>Mutasim Abubakar</p>

**Comment by airtower-luna** [comment]
<p>You’ll have to create a ticket with <a href="https://support.github.com/request" class="inline-onebox">GitHub Support</a>. Us other users can’t help there.</p>
